code/ai/ai_profiles.cpp

```diff
@@ -581,6 +581,16 @@ void parse_ai_profiles_tbl(const char *filename)
 
 				set_flag(profile, "$reset last_hit_target_time for player hits:", AI::Profile_Flags::Reset_last_hit_target_time_for_player_hits);
 
+				if (optional_string("$Turret target recheck time:"))
+				{
+					stuff_float(&profile->turret_target_recheck_time);
+					if (profile->turret_target_recheck_time < 0) {
```

Comment:
Is there a sensible minimum value we want to enforce for the recheck time?

Reply:
Theoretically, being just a timestamp, anything >= 0 is logically valid and I haven't seen any issues in my test cases with 0. But practically, EatThePath using this with 0 delay resulted in frequent stuttering (and not just poor framerate), which is a bit unusual to me as I would've expected merely low framerate, and recommended 15-30ms as a minimum for no stuttering. Ideally though, I would like to not limit this unless there are actual logic errors with it, and leave potential performance concerns to warning labels on the wiki documentation.

Reply:
Fair enough.